components/gitpod-cli/cmd/prebuild-logs.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/theialib"
```

Comment:
We should ditch theialib. The official api is supervisor API, maybe we should think about how enrich it to return task output. We were discussing such API in the past with @geropl that dashboard could as well return output of prebuild task even if some task already finished. 

I think it would be in the right direction to make prebuild UX is more reliable. cc @csweichel